src/components/ScenarioParameters/components/ScenarioParametersTabs/ScenarioParametersTabsWrapper.js

```diff
@@ -43,22 +37,13 @@ const ScenarioParametersTabsWrapper = ({
     if ('name' in tabFactory || (tabFactory?.WrappedComponent && 'name' in tabFactory.WrappedComponent)) {
       parametersGroupMetadata.tab = React.createElement(tabFactory, {
         parametersGroupData: parametersGroupMetadata,
-        parametersState: parametersValuesToRender,
-        setParametersState: setParametersValuesToRender,
         context,
       });
     }
     // Note that the factories are now deprecated and
     // won't be supported in the next major version of the webapp
     else {
-      parametersGroupMetadata.tab = tabFactory.create(
-        t,
-        datasets,
-        parametersGroupMetadata,
-        parametersValuesToRender,
-        setParametersValuesToRender,
-        context
-      );
```

Comment:
This change is breaking the compatibility to "old" factories (deprecated since v3.0), we should either keep it untouched, or we could also drop the support of the deprecated "factories-as-functions" mechanism.
Note: we don't have any tests to check the deprecated factories syntax, we could probably drop their support now

Reply:
done